Source/Mock.cs

```diff
@@ -261,8 +265,10 @@ public void Verify()
 
 		private bool TryVerify(out UnmatchedSetups error)
 		{
-			if (!this.Interceptor.TryVerify(out error))
+			var uninvokedVerifiableSetups = this.Setups.ToArrayLive(setup => setup.IsVerifiable && !setup.Invoked);
```

Comment:
By moving setups from `InterceptorContext` into `Mock`, verification is no longer spread across three types (`Mock`, `Interceptor`, `InterceptorContext`) but happens completely within `Mock`.